README.md

```diff
@@ -19,7 +19,9 @@ npm install cjs-module-lexer
 For use in CommonJS:
 
 ```js
-const parse = require('cjs-module-lexer');
+const { parse, init } = require('cjs-module-lexer');
```

Comment:
not sure if we should require it to be called or not. E.g. what happens if I use `import('cjs-module-lexer')`? Possibly confusing. Dunno 🙂

Reply:
since we're changing the API regardless it might be worth it. happy to make the change if you want 👍

Reply:
Strictly speaking, the way the exports are defined in https://github.com/guybedford/cjs-module-lexer/blob/master/package.json#L6, `import()` should always get the ES module form by definition of the resolution rules and even in bundlers (which RollupJS and Webpack support now!)